WebApp/public/scripts/register-events.js

```diff
@@ -337,18 +373,54 @@ export function registerMouseEvents(videoPlayer, playerElement) {
   }
 
   function sendMouseWheel(e) {
-    console.log("mouse wheel with delta " + e.wheelDelta);
+    //console.log("mouse wheel with delta " + e.wheelDelta);
     let data = new DataView(new ArrayBuffer(9));
     data.setUint8(0, InputEvent.MouseWheel);
     data.setFloat32(1, e.deltaX, true);
     data.setFloat32(5, e.deltaY, true);
     _videoPlayer && _videoPlayer.sendMsg(data.buffer);
   }
+
+    // Listen to mouse events
+    playerElement.addEventListener('click', sendMouse, false);
+    playerElement.addEventListener('mousedown', sendMouse, false);
+    playerElement.addEventListener('mouseup', sendMouse, false);
+    playerElement.addEventListener('mousemove', sendMouse, false);
+    playerElement.addEventListener('wheel', sendMouseWheel, false);
+
+    // Listen to touch events based on "Touch Events Level1" TR.
+    //
+    // Touch event Level1 https://www.w3.org/TR/touch-events/
+    // Touch event Level2 https://w3c.github.io/touch-events/
+    //
+    playerElement.addEventListener('touchend', sendTouchEnd, false);
+    playerElement.addEventListener('touchstart', sendTouchStart, false);
+    playerElement.addEventListener('touchcancel', sendTouchCancel, false);
+    playerElement.addEventListener('touchmove', sendTouchMove, false);
+}
+
+
+export function unregisterMouseEvents(videoPlayer, playerElement) {

```

Comment:
Please fix indents  (4space -> 2space)

Reply:
I'm using Visual Studio to work, for this project and a lot of others, I don't want to change the default standard of automatic 4 space indenting just for this file. I don't think this is a real issue